packages/react/src/field/Field.tsx

```diff
@@ -42,17 +48,31 @@ export const Field = forwardRef<HTMLDivElement, FieldProps>(
     return (
       <Flex flexDirection="column" gap="2" maxW="sm" ref={ref} {...props}>
         {label && (
-          <Text asChild>
-            <RadixLabel.Root htmlFor={id}>
-              {label}
-              {required && (
-                <Text aria-hidden="true" as="span" color="fg.error">
-                  {" "}
-                  *
-                </Text>
-              )}
-            </RadixLabel.Root>
-          </Text>
+          <Flex flexDirection="row" gap="4">
+            <Text
+              alignItems="center"
+              asChild
+              display="flex"
+              fontWeight="500"
+              gap="2"
+            >
+              <RadixLabel.Root htmlFor={id}>
```

Comment:
let's invert this please - so the radix or external component should always come first and inside it should be our component
```suggestion
            <RadixLabel.Root asChild htmlFor={id}>
              <Text
                alignItems="center"
                display="flex"
                fontWeight="500"
                gap="2"
              >
```